### PR TITLE
Remove contributor stuff from PDF user guide

### PR DESCRIPTION
**Pull request by trey-stafford**
## Description

We don't want contributor docs in the User Guide PDF. Apply a hack to resolve.

## Checklist

If an item on this list is done _or not needed_, simply check it with `[x]`.

- [ ] Config lockfile updated (`inv config.export > qgreenland/config/cfg-lock.json`)
- [ ] Version bumped if needed (`bumpversion (major|minor|patch|prerelease|build`)
- [ ] CHANGELOG.md updated
- [ ] Documentation updated if needed
- [ ] New unit tests if needed

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -125,42 +123,50 @@ def linkcode_resolve(domain, info):
...
-# Hide redundant field summary
-autodoc_pydantic_model_show_field_summary = False
-
-# Don't show collapsible JSONSchema model. It's too big for the PDF output. Is
-# there a way to enable it for HTML only and disable for PDF?
-autodoc_pydantic_model_show_json = False
-
-# Don't warn when a field is not serializable
-autodoc_pydantic_model_show_json_error_strategy = "coerce"
+if 'latexpdf' not in sys.argv:
```
Comment (trey-stafford) [inline doc/conf.py:126]: hacky

Comment (mfisher87) [reply doc/conf.py:126]: We need a comment explaining the why of the hack, and maybe a TODO explaining how we think we can un-hack :)

**Review by mfisher87 — CHANGES_REQUESTED** [review]

File: doc/index.rst [hunk 2]
```diff
@@ -100,11 +110,6 @@ QGreenland Documentation
     discussion/*
 
 
-.. raw:: latex
```
[the remark this answers is not in the document]
Comment (trey-stafford) [reply doc/index.rst:103]: I think we originally added this appendix page to the pdf to separate the python package index added by autodoc. There's now no content for the appendix in the produced pdf.

**Comment by trey-stafford** [comment]
We should probably also remove other contributor documentation. For example, the discussion section has a section for contributing to qgreenland that makes references to our code on github.

**Review by MattF-NSIDC — APPROVED** [review]

**Comment by MattF-NSIDC** [comment]
Resolves #524

**Comment by MattF-NSIDC** [comment]
OBE, resolved by #523

**Pull request closed without merging**